plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/TestDeltaLakeLegacyCreateTableWithExistingLocationUsingConfigProperty.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.deltalake;
+
+import com.google.common.collect.ImmutableMap;
+import io.trino.Session;
+
+import java.util.Map;
+
+import static io.trino.testing.TestingSession.testSessionBuilder;
+
+public class TestDeltaLakeLegacyCreateTableWithExistingLocationUsingConfigProperty
```

Comment:
You don't need to verify that session properties get initalized with configu properties.  That's nice, but also expensive to setup a new distributed query runner just for that

Reply:
Removed it.